README.md

```diff
@@ -3,24 +3,22 @@
 >Warning: Experimental and still under development. Not meant for production deployment.
 >Note: This component is currently only functional with Istio as the networking layer.
 
-Asynchronous component to enable async Knative service calls.
+This is an add-on component that, when installed, will enable your Knative services to be called asynchronously. You can set a service to be always or conditionally asynchronous. Conditionally asynchronous services will respond when the `Prefer: respond-async` header is provided as a part of the request.
 
 ![diagram](./README-images/diagram.png)
 
 ## Install Knative Serving & Eventing to your Cluster
 
 1. https://knative.dev/docs/install/any-kubernetes-cluster/
 
-## Create your Demo Application.
+## Create your demo application
 
-1. This can be, at a minimum, a simple hello world application that sleeps for some time.
-   There is a sample application that sleeps for 10 seconds in the `test/app`
-   folder. To deploy, use the `kubectl apply` command:
+1. This can be any simple hello world application. There is a sample application that sleeps for 10 seconds in the `test/app` folder. To deploy, use the `kubectl apply` command:
```

Comment:
I would then link the helloworld-go Knative app: https://knative.dev/docs/serving/samples/hello-world/helloworld-go/

Reply:
updated, thanks!